vpc/service/metrics/metrics.go

```diff
@@ -17,10 +17,11 @@ var (
 	connectionsIdle    = stats.Int64("db.connectionsIdle", "The number of idle connections", "connections")
 
 	// Counter measures
-	waitCount         = stats.Int64("db.waitCount", "The total number of connections waited for", "connections")
-	waitDuration      = stats.Int64("db.waitDuration", "The total time blocked waiting for a new connection", "ns")
-	maxIdleClosed     = stats.Int64("db.maxIdleClosed", "The total number of connections closed due to SetMaxIdleConns", "connections")
-	maxLifetimeClosed = stats.Int64("db.maxLifetimeClosed", "The total number of connections closed due to SetConnMaxLifetime", "connections")
+	waitCount               = stats.Int64("db.waitCount", "The total number of connections waited for", "connections")
+	waitDuration            = stats.Int64("db.waitDuration", "The total time blocked waiting for a new connection", "ns")
+	maxIdleClosed           = stats.Int64("db.maxIdleClosed", "The total number of connections closed due to SetMaxIdleConns", "connections")
+	maxLifetimeClosed       = stats.Int64("db.maxLifetimeClosed", "The total number of connections closed due to SetConnMaxLifetime", "connections")
+	ErrorScanBranchEniCount = stats.Int64("error.scanBranchEni", "The total number of connections closed due to SetConnMaxLifetime", stats.UnitNone)
```

Comment:
"ENI's that could not be cleaned up due to error in query"?

Reply:
Oops copy-pasta.